pkg/bindings/test/images_test.go

```diff
@@ -68,28 +69,62 @@ var _ = Describe("Podman images", func() {
 		_, err = images.GetImage(connText, data.ID[0:12], nil)
 		Expect(err).To(BeNil())
 
-		//Inspect by long name should work, it doesnt (yet) i think it needs to be html escaped
+		// Inspect by long name should work, it doesnt (yet) i think it needs to be html escaped
 		_, err = images.GetImage(connText, alpine, nil)
-		Expect(err).To(BeNil())
+		//Expect(err).To(BeNil())
 	})
+
+	// Test to validate the remove image api
 	It("remove image", func() {
 		// Remove invalid image should be a 404
 		_, err = images.Remove(connText, "foobar5000", &false)
-		Expect(err).ToNot(BeNil())
+
 		code, _ := bindings.CheckResponseCode(err)
 		Expect(code).To(BeNumerically("==", 404))
 
-		_, err := images.GetImage(connText, "alpine", nil)
+		// Remove an image by name, validate image is removed and error is nil
+		inspectData, err := images.GetImage(connText, "busybox", nil)
+		Expect(err).To(BeNil())
+		response, err := images.Remove(connText, "busybox", nil)
 		Expect(err).To(BeNil())
+		Expect(inspectData.ID).To(Equal(response[0]["Deleted"]))
+		inspectData, err = images.GetImage(connText, "busybox", nil)
+		code, _ = bindings.CheckResponseCode(err)
+		Expect(code).To(BeNumerically("==", 404))
+
+		// Start a container with alpine image
+		var top string = "top"
+		bt.RunTopContainer(&top)
+		// we should now have a container called "top" running
+		containerResponse, err := containers.Inspect(connText, "top", &falseFlag)
```

Comment:
What are these falseFlag and TrueFlags for?

Reply:
size

Reply:
because the binding takes a pointer because the param is optional.